src/utils.tsx

```diff
@@ -107,9 +107,51 @@ const emojiMarkdownPlugin = () => {
   return transform;
 };
 
-export const renderText = <Us extends DefaultUserType<Us> = DefaultUserType>(
+type MentionedUser<
+  Us extends DefaultUserType<Us> = DefaultUserType
+> = UserResponse<Us>;
```

Comment:
we shouldn't need this type, because in order to actually use it with the generic you'd have to call it like MentionedUser<Us>, so it would be identical to UserResponse<Us>